log/log_redirect_arm64.go

```diff
@@ -0,0 +1,23 @@
+package log
```

Comment:
I think this should include a build flag as well to only build on arm64?

Reply:
As quoted from https://golang.org/pkg/go/build/#hdr-Build_Constraints

> If a file's name, after stripping the extension and a possible _test suffix, matches any of the following patterns:
> 
> ```
> *_GOOS
> *_GOARCH
> *_GOOS_GOARCH
> ```
> 
> (example: source_windows_amd64.go) where GOOS and GOARCH represent any known operating system and architecture values respectively, then the file is considered to have an implicit build constraint requiring those terms (in addition to any explicit constraints in the file).